src/core/i18n/translate.js

```diff
@@ -41,10 +41,6 @@ export default function translate(options = {}) {
       i18n: PropTypes.object.isRequired,
```

Comment:
Can you remove this too ^ ? This looks like a left over from when the I18nProvider parts were extracted/refactored. 

I wonder if i18nProvider could possibly be ultimately removed if it's not used that much following what was started in https://github.com/mozilla/addons-frontend/issues/1262 - if not it might need updating to the newer context API if it still requires context.

Reply:
mmh don't we need this to tell the parent component we want the `i18n` object?

I will look at the I18Provider.